s4e-web/cypress/integration/user-options/user-options-mail-to-expert.spec.ts

```diff
@@ -15,12 +15,12 @@ describe.skip("Map Options", () => {
   });
 
   it("Should send mail to expert", () => {
-    MapOptions
+    MailToExpert
       .openSendMailToExpertModal()
       .addMessageToSupport('Wsparcie zdalne',"Test Message")
       .sendMessageToSupport()
       .changeContextTo(ConfirmModal)
-      .acceptAndChangeContextTo(MapOptions)
+      .acceptAndChangeContextTo(MailToExpert)
```

Comment:
We should drop context change idea in functions. Please replace the piece of code with

```
...
.accept();
MailToExpert
          .confirmationShouldToAppear()

Reply:
this will be change in next PR